http/server.go

```
@@ -1208,54 +854,28 @@ func (self *HTTPServer) GetTradeHistory(c *gin.Context) {
 	for _, ex := range self.exchanges {
 		history, err := ex.GetTradeHistory(fromTime, toTime)
 		if err != nil {
-			c.JSON(
-				http.StatusOK,
-				gin.H{
-					"success": false,
-					"data":    err.Error(),
-				},
-			)
+			// TODO: response should include error in reason field instead of data?
```

@halink0803 could you help me with this TODO? As I read the rest, it should return the error value in reason key instead of data.

---

I changed it. It should be return reason instead of data.